instance/tests/test_tasks.py

```diff
@@ -77,7 +82,38 @@ def test_watch_pr_new(self, mock_get_username_list, mock_get_pr_list_from_userna
         self.assertEqual(instance.fork_name, 'watched/fork')
         self.assertEqual(instance.github_pr_number, 234)
         self.assertEqual(instance.branch_name, 'watch-branch')
-        self.assertEqual(instance.ansible_extra_settings, 'WATCH: true\r\n')
+        self.assertEqual(instance.database_url, 'mysql://db.opencraft.com')
+        self.assertEqual(instance.mongo_url, 'mongo://mongo.opencraft.com')
+        self.assertEqual(yaml.load(instance.ansible_extra_settings), {'WATCH': True})
         self.assertEqual(
             instance.name,
             'PR#234: Watched PR title which ... (bradenmacdonald) - watched/watch-branch (7777777)')
+
+    @override_settings(INSTANCE_DATABASE_URL='mysql://db.opencraft.com',
+                       INSTANCE_MONGO_URL='mongo://mongo.opencraft.com')
+    @patch('instance.models.instance.github.get_commit_id_from_ref')
+    @patch('instance.tasks.provision_instance')
+    @patch('instance.tasks.get_pr_list_from_username')
+    @patch('instance.tasks.get_username_list_from_team')
+    def test_watch_pr_ephemeral_databases(self, mock_get_username_list,
+                                          mock_get_pr_list_from_username,
+                                          mock_provision_instance,
+                                          mock_get_commit_id_from_ref):
+        """
+        Asking for ephemeral databases in the PR should unset external
+        database settings, even if the default is to use external databases
+        """
+        mock_get_username_list.return_value = ['itsjeyd']
+        settings = 'EPHEMERAL_DATABASES: yes\r\n'
```

Comment:
@omarkhan Yup - it will need to be reformatted a little to be included in the repo, to be made more generic, but in the meantime I copied the current doc there: https://gist.github.com/antoviaque/8f989565138f05d08aaf